react-native-aztec/src/utils.js

```diff
@@ -0,0 +1,12 @@
+import { Platform } from 'react-native';
+
+// Basically React Native has a mechanism to scale font size automatically.
+// But GM is rendered on top of fragment of which retain flag is true.
+// Hence we deal with fontScale change manually on our side.
+export function getScaledFontSize( fontSize, fontScale ) {
+	if ( Platform.OS === 'ios' ) {
+		return fontSize ? fontSize : undefined; // to not to affect to iOS
+	}
+	const baseFontSize = 18;
```

Comment:
Should we use the default value set in RN for this? https://github.com/facebook/react-native/blob/1151c096dab17e5d9a6ac05b61aacecd4305f3db/ReactAndroid/src/main/java/com/facebook/react/uimanager/ViewDefaults.java#L15

Reply:
This is a bit tricky but I did reverse-engineering to find out 18 is the base fontSize for paragraph block. If we use 14 you'll see the base font size is smaller than before.

| no fontSize specify | baseFontSize=18 |
| ----------- | ----------- |
|<img src="https://user-images.githubusercontent.com/471318/55343224-a096eb80-54e5-11e9-9fa6-d44c5cc66448.png" width="250" />|<img src="https://user-images.githubusercontent.com/471318/55343273-b3a9bb80-54e5-11e9-81c6-36160cf9ff05.png" width="250" />|

But yea it's too tricky...